src/bond/assertions.clj

```diff
@@ -17,3 +17,19 @@
   `vargs` should be a vector of args [args-first-call args-second-call ...] to allow for the checking of multiple calls of `f`."
   [f vargs]
   (= vargs (->> f bond/calls (mapv :args))))
+
+(defn called-once-with-args?
+  "An assertion to check if `f` was called with `args` strictly once.
+   
+   `args` should be a vector/coll of args [arg1 arg2 arg3] to compare directly to the value of `:args` from `bond/calls`"
+  [f args]
+  (and (called-times? f 1)
+       (= args (-> f bond/calls first :args))))
```

Comment:
this be written using `called-with-args?` i think? `(called-with-args? f [args])`